attractors/attractors.ipynb

```diff
@@ -95,6 +95,16 @@
     "We can now aggregate these 10,000,000 continuous coordinates into a discrete 2D rectangular grid with [Datashader](http://datashader.org), counting each time a point fell into that grid cell:"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import datashader as ds\n",
+    "from datashader import transfer_functions as tf"
```

Comment:
Most of what's in `transfer_functions` is also in the main datashader namespace. Can you please try replacing `tf.` with `ds.` and if that all works, delete this import?  I think it's clearer not to bring up "transfer functions" for these users.

Reply:
I tried it and it didn't work.

Reply:
Oops; I steered you wrong: replace `tf.` with `ds.tf.`. `tf` is already imported in to the `ds` namespace, and that way the code is shorter and you don't have to explain "transfer functions". Of course, `.tf` is still mysterious; it's just short enough that I don't think it will catch people.